ros2cli_test_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.5)
+project(ros2cli_test_msgs)
```

Comment:
nitpick: I prefer the suffix `_interfaces` in the package name, since the package contains more than just messages (ie. also actions and services).

Reply:
Sounds good, I think interfaces is better, too - I was following the example of `test_msgs`, which also has services and actions.

Reply:
Yeah, earlier packages in the development of ROS 2 follow the ROS 1 naming convention with the `_msgs` suffix.